src/test/java/com/mongodb/jdbc/MongoDatabaseMetaDataTest.java

```diff
@@ -400,4 +404,12 @@ void testGetSchemas() throws SQLException {
         rs = databaseMetaData.getSchemas(null, null);
         validateResultSet(rs, 0, columns);
     }
+
+    @Test
+    @Override
+    void testGetFunctions() throws SQLException {
+        testGetFunctionsHelper("%", 16);
```

Comment:
This is nice and concise.  I see it is testing the different function name patterns, but not super clear as to what it's testing.  Please add a brief comment on what is being tested.